cloud-governance-mcp/run_agent.sh

```
@@ -0,0 +1,63 @@
+# Activate virtual environment
+echo "Activating virtual environment..."
+source .venv/bin/activate
+
+# Install/update dependencies
+echo "Installing dependencies..."
+python -m pip install -U pip > /dev/null
+python -m pip install -r requirements.txt
+
+# Kill existing Streamlit processes on port 8501
+echo "Stopping existing Streamlit processes..."
+kill -9 $(lsof -ti tcp:8501) 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)